### PR TITLE
Add the es_sink library to the community repo

### PR DESCRIPTION
**Pull request by Jon-AtAWS**
*Issue #, if available:*
None

*Description of changes:*
Adds the es_sink package to the community repo. Some changes to the CFN templates to update to ODFE 1.2.1

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: es_sink/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (yojs) [inline es_sink/LICENSE]: Does this make this package dual licensed under both GPVv2 and MIT ? Is this intended ?

File: es_sink/setup.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```
Comment (yojs) [inline es_sink/setup.py]: License headers ?

Comment (Jon-AtAWS) [reply es_sink/setup.py]: AFAIK, this is fine. Let me check whether MIT works or I need to change to Apache

Comment (alolita) [reply es_sink/setup.py]: MIT works - compatible w Apache 2

**Pull request merged**